modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -273,6 +274,7 @@ public ObjectDefinition addOrUpdateSystemObjectDefinition(
 			return objectDefinition;
 		}
 
+		objectDefinition.setObjectFolderId(objectFolderId);
```

Comment:
I didn't see necessity to validate the ID because this method is only used in the [SystemObjectDefinitionManagerPortalInstanceLifecycleListener](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-service/src/main/java/com/liferay/object/internal/instance/lifecycle/SystemObjectDefinitionManagerPortalInstanceLifecycleListener.java#L179-L181). 
In this case, we will always make sure that the ID sent is valid. Do you think better add a validation?